assets/src/edit-story/components/library/panes/media/media3p/api/index.js

```diff
@@ -0,0 +1,18 @@
+/*
```

Comment:
Perhaps a better folder for this is closer to the state mgt such as ```src/edit-story/app/media/media3p/api```, since the media pane won't reference useMedia3pApiProvider directly.